metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Taxonomy.java

```diff
@@ -178,4 +178,9 @@ public void setTaxonomyForRank(Taxonomy taxonomy, String rank) {
                 break;
         }
     }
+
+    @Override
+    public String getStudyIds() {
+        return null;
+    }
```

Comment:
Could you please also document all cases where `null` is returned with something like “Access to entities of this kind is not restricted, hence the special value of `null` is returned”